packages/core/src/render3/instructions.ts

```diff
@@ -541,7 +539,8 @@ function createStaticData(
     attrs,
     initialInputs: undefined,
     inputs: undefined,
-    outputs: undefined, containerStatic
+    outputs: undefined,
+    containerStatic
```

Comment:
I am not a fan of this short hand since renaming local `containerStatic` will break API. Could we write `containerStatic: containerStatic`